src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1554805302.inc.php

```diff
@@ -0,0 +1,53 @@
+<h1>Build #1554805302</h1>
+<h2>Date: 2019-04-09</h2>
+<div class="changelog">
+    - Hide icon field in table and module settings
+</div>
+<?php
+
+$fieldId = TCMSLogChange::GetTableFieldId(TCMSLogChange::GetTableId('cms_tbl_conf'), 'icon_list');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'modifier' => 'hidden',
```

Comment:
"hidden" is enough to not require it?

I would expect an "is_required=0".

Reply:
Nope, hidden is enough.